lib/ui/pages/pages.dart

```diff
@@ -1,10 +1,11 @@
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:food_app/ui/widgets/widgets.dart';
-import 'package:food_app/utils/app_router.dart';
 import 'package:food_app/utils/app_typography.dart';
 import 'package:food_app/utils/app_color.dart';
 
```

Comment:
we supposed to not removed app_router import because we have still use it in login_page.dart and signup_page.dart

```suggestion
import 'package:food_app/utils/app_router.dart';
```